templates/corporate/features.html

```diff
@@ -1,380 +1,27 @@
-{% extends "zerver/portico.html" %}
-{% set entrypoint = "landing-page" %}
+{% extends "zerver/base.html" %}
+{% set entrypoint = "plans-page" %}
```

Comment:
Not a thing for this PR, but we might want to discuss in #frontend doing a batch of renaming/reorganization, like having a `legacy_landing_pages.css` and `legacy-landing-pages` bundle, just to make it easier to follow.